### ISSUE TITLE
Accessing the database during app initialization

### ISSUE THREAD

**Issue by spapas** [chrisclark/django-sql-explorer#612]
After ugprading to the latest version (django-sql-explorer==4.1) I had to install sql-metadata (see #610), however after that I'm getting this warning:

```
C:\progr\py3\ehcg\venv\lib\site-packages\django\db\backends\utils.py:98: RuntimeWarning: Accessing the database during app initialization is discouraged. To fix this warning, avoid executing queries in AppConfig.ready() or when your app modules are imported.
```

The warning is from the `track_summary_stats` that you have on the `ExplorerAppConfig`.

**Comment by chrisclark** [comment]
Interesting. That is new in Django 5 and I didn't catch in testing. I will address alongside the other 4.1 issues and push a 4.2. This will require a new model to store a high-water-mark setting.

**Comment by chrisclark** [comment]
Addressed in #611
